services/src/northbound/src/test/java/org/openkilda/northbound/controller/TestMessageMock.java

```diff
@@ -83,11 +85,13 @@
             SWITCH_ID, 1, 1, 1, 1, 1, 1, path, FlowState.UP);
 
     private static final FlowResponse flowResponse = new FlowResponse(flowModel);
-    private static final FlowReadResponse FLOW_RESPONSE =
+    static final FlowReadResponse FLOW_RESPONSE =
             new FlowReadResponse(new BidirectionalFlow(flowModel, flowModel));
     private static final SwitchRulesResponse switchRulesResponse =
             new SwitchRulesResponse(singletonList(TEST_SWITCH_RULE_COOKIE));
     private static final Map<String, CommandData> messages = new ConcurrentHashMap<>();
+    static final SwitchInfoData SWITCH_INFO_DATA =
+            new SwitchInfoData(SWITCH_ID, SwitchState.ACTIVATED, "", "", "", "");
```

Comment:
Is it ok if we pass an address, a hostname, a description and a controller as `""`?